pkg/attestation/crafter/runners/azurepipeline.go

```diff
@@ -83,3 +84,11 @@ func (r *AzurePipeline) RunURI() (url string) {
 func (r *AzurePipeline) ResolveEnvVars() (map[string]string, []*error) {
 	return resolveEnvVars(r.ListEnvVars())
 }
+
+func (r *AzurePipeline) WorkflowFile(_ context.Context) string {
+	return ""
+}
+
+func (r *AzurePipeline) IsHosted(_ context.Context) bool {
+	return false
```

Comment:
Shouldn't it be `true` by default for all hosted CIs? (azure, github, gitlab, jenkins, circleci ...)

Reply:
No, this shouldn't be `true` by default, in fact we should only set it to `true` if we are sure that this is hosted. Based on the current approach, so using environment variables ain't enough IMHO to say that something is hosted. Maybe we should rename this, for example to something like authenticated or something?

Reply:
Actually, I went with the suggestion from @migmartri  and now we have `IsHosted` and `IsAuthenticated` - the `IsHosted` will be true for the defined runners, but the `IsAuthenticated` will be `true` only for GitHub Actions now where the OIDC token will be properly validated.